crud/common/map_call_cases/batch_postprocessor.lua

```diff
@@ -39,6 +39,11 @@ function BatchPostprocessor:collect(result_info, err_info)
         wrapper_args = '?table',
     })
 
+    if result_info.value ~= nil then
+        self.storage_info[result_info.key] = {}
+        self.storage_info[result_info.key].replica_schema_version = result_info.value[3]
```

Comment:
Why not perform it in a single step?

Reply:
AFAIU, need to create table on key with uuid=`result_info.key`, else without it:

```lua
crud/common/map_call_cases/batch_postprocessor.lua:44: attempt to index a nil value
```

Reply:
Done